docs/copilot/language-models.md

```diff
@@ -43,6 +43,25 @@ You can further extend the list of available models by [using your own language
 
 If you have a paid Copilot plan, the model picker shows the premium request multiplier for premium models. Learn more about [premium requests](https://docs.github.com/en/copilot/managing-copilot/monitoring-usage-and-entitlements/about-premium-requests#premium-requests) in the GitHub Copilot documentation.
 
+## Auto model selection
+
+> [!NOTE]
+> Auto model selection is available as of VS Code release 1.104 and is currently only available in [Insiders](https://code.visualstudio.com/insiders/).
+
+With auto model selection, VS Code automatically selects the best model for your task based on available capacity. This ensures that you get the best performance and helps avoid rate limiting due to excessive usage of particular language models. We continue to improve this feature to use more advanced criteria for model selection.
```

Comment:
I would add one more sentence on the benefit. Something like "Auto will detect degraded model performance and pick the best model at that point in time."